nixos/tests/custom-ca.nix

```diff
@@ -153,17 +151,21 @@ in
         machine.fail("curl -fv https://bad.example.com")
 
     browsers = [
-      # Firefox was disabled here, because we needed to disable p11-kit support in nss,
-      # which is why it will not use the system certificate store for the time being.
-      # "firefox",
+      "firefox",
       "chromium",
-      "falkon",
+      # "falkon",  # Falkon is broken because kwallet is broken.
```

Comment:
This is ironic: I had picked falkon because I was assuming it was the most stable.
You should be able to replace it with `qutebrowser` and the error text is `Certificate error`.